ci-operator/step-registry/ibmcloud/provision/dedicatedhost/ibmcloud-provision-dedicatedhost-commands.sh

```diff
@@ -0,0 +1,217 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+#Since sometimes fail to create the dedicated host in some zone, so just continue test when create dedicated host succeed, and create cluster just on one zone. 
+#Known Issues: 
+# OCPBUGS-5906 [IPI-IBMCloud] the worker nodes are “Provisioning” when using the dedicated host in the Default group
+# OCPBUGS-18925 [IPI-IBMCloud] ] fail to use vx2d-host-176x2464 profile with dedicated host in install-config
+
+function run_command() {
+    local CMD="$1"
+    echo "Running Command: ${CMD}"
+    eval "${CMD}"
+}
+
+# IBM Cloud CLI login
+function ibmcloud_login {
+    export IBMCLOUD_CLI=ibmcloud
+    export IBMCLOUD_HOME=/output
+    region="${LEASED_RESOURCE}"
+    export region
+    echo "Try to login..."
+    "${IBMCLOUD_CLI}" login -r ${region} --apikey @"${CLUSTER_PROFILE_DIR}/ibmcloud-api-key"
+}
+
+
+function create_resource_group() {
+    local rg="$1"
+    echo "create resource group ... ${rg}"
+    "${IBMCLOUD_CLI}" resource group-create ${rg} || return 1
+    echo "${rg}" > "${SHARED_DIR}/ibmcloud_resource_group"
+    "${IBMCLOUD_CLI}" target -g ${rg} || return 1
+}
+
+#random select the group which create the dedicatd host
+function randomSelectRG {
+    dhGroups=("Default" "${resource_group}")
+    selected_idx=$((RANDOM % 2))
+    rg=${dhGroups[selected_idx]}
+    echo $rg
+}
+
+function randomGetDHProfile {
+    local all_profiles profiles selected_id proTotal profile tmp family i
+    all_profiles=$(${IBMCLOUD_CLI} is dhps -q | awk '(NR>1) {print $1}')
```

Comment:
This command line can easily get all the wanted profile name instead of the following complicated shell script.
```
ibmcloud is dhps --output JSON | jq -r '.[] | select(.family != "very-high-memory") | .name'
```

Reply:
not supporting "very-high-memory"  is a bug just found in testing this PR. Thanks for your suggestion.